tests/test_api.py

```diff
@@ -1281,6 +1285,57 @@ def test_submit_manifest_table_and_file_upsert(
         )
         assert response_csv.status_code == 200
 
+    @pytest.mark.synapse_credentials_needed
+    @pytest.mark.submission
+    def test_submit_nested_manifest_table_and_file_replace(
+        self,
+        client: FlaskClient,
+        request_headers: Dict[str, str],
+        helpers: Helpers,
+        synapse_store: SynapseStorage,
+    ) -> None:
+        # GIVEN the parameters to submit a manifest
+        params = {
+            "schema_url": DATA_MODEL_JSON_LD,
+            "data_type": "BulkRNA-seqAssay",
+            "restrict_rules": False,
+            "manifest_record_type": "table_and_file",
+            "asset_view": "syn63646213",
+            "dataset_id": "syn63646197",
+            "table_manipulation": "replace",
+            "data_model_labels": "class_label",
+            "table_column_names": "display_name",
+        }
+
+        # AND a test manifest with a nested file entity
+        test_upsert_manifest_path = helpers.get_data_path(
```

Comment:
Along the lines of @linglp's comment, this is still a table replace operation right?

Reply:
This was addressed